Rakefile

```diff
@@ -1,2 +1,3 @@
 raise "only JRuby is supported" unless defined? JRUBY_VERSION
 require 'bundler/gem_tasks'
```

Comment:
do we also want to keep `rake release` etc if we're mean to use `rake publish_gem` ?
(OT but it was very confusing at start that LS gem releases do not follow the "ruby" standard)